File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 4 - C/q6.c
//. Faça um algoritmo que receba três números e armazene-os em três variáveis com os seguintes
//nomes: maior, intermediário e menor. Depois imprima na tela os valores.

#include<stdio.h>
int main(){
	int a, b, c;
	printf("Insira o valor A: ");
	scanf("%d", &a);
	printf("Insira o valor B: ");
	scanf("%d", &b);
	printf("Insira o valor C: ");
	scanf("%d", &c);
	if(a>b && b>c){
		printf("Menor: %d\nIntermediario: %d\nMaior %d", c,b,a);
	}
	else if (b>c && c>a){
		printf("Menor: %d\nIntermediario: %d\nMaior %d", a,c,b); 
	}
	else if (c>b && b>a){
		printf("Menor: %d\nIntermediario: %d\nMaior %d", a,b,c);
	}
	else if (b>a && a>c){
		printf("Menor: %d\nIntermediario: %d\nMaior %d", c,a,b);
	}
	else if (a>c && c>b){
		printf("Menor: %d\nIntermediario: %d\nMaior %d", b,c,a);
	}	
	else if (c>a && a>b){
		printf("Menor: %d\nIntermediario: %d\nMaior %d", b,a,c);
	}
	system("pause");		
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 4 - Python/q6.py
#Faça um algoritmo que receba três números e armazene-os em três variáveis com os seguintes
#nomes: maior, intermediário e menor. Depois imprima na tela os valores.

x=int(input("Insira o número: "))
y=int(input("Insira o número: "))
z=int(input("Insira o número: "))
if x>y and y>z:
    print("Maior: ", x, "\nIntermediário: ", y, "\nMenor: ", z)
elif y>z and z>x:
    print("Maior: ", y, "\nIntermediário: ", z, "\nMenor: ", x)
elif z>y and y>x:
    print("Maior: ", z, "\nIntermediário: ", y, "\nMenor: ", x) 
elif y>x and x>z:
    print("Maior: ", y, "\nIntermediário: ", x, "\nMenor: ", z)
elif x>z and z>y:
    print("Maior: ", x, "\nIntermediário: ", z, "\nMenor: ", y)
elif z>x and x>y:
    print("Maior: ", z, "\nIntermediário: ", x, "\nMenor: ", y)
